### PR TITLE
Fix accidental caching

### PR DESCRIPTION
**Pull request by asilluron**
@mathiasbynens caught an issue where a user would see details from another session. See: https://i.imgur.com/XJqs3DT.png

The root cause is that _assign from lodash mutates. "Note: This method mutates object".

### CODE REVIEW

**Comment by maxbeatty** [comment]
I know conceptually this is the fix. Can we prove it with a test so it stays this way?

**Comment by asilluron** [comment]
Absolutely

File: server/web/home/index.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ exports.register = function (server, options, next) {
         authorized = true;
       }
 
-      reply.view('home/index', _assign(defaults.testPageContext, {
+      reply.view('home/index', Hoek.applyToDefaults(defaults.testPageContext, {
```
Comment (mathiasbynens) [inline server/web/home/index.js]: Or, we could stick to `_assign` but use `_assign({}, defaults.testPageContext, {…})`

**Comment by asilluron** [comment]
Dang thought the test would be a breeze

```
server.inject(request, function (response) {
          server.inject({
            method: 'GET',
            url: '/'
          }, function (resp) {
            Code.expect(resp.result).to.not.include('Pitcher Man');

            done();
          });
        });
```

But **hapi/shot** starts up the route handler from scratch each time. So I am having to do this a more clever way.

File: server/web/home/index.js (hunk 1 above)
Comment (asilluron) [inline server/web/home/index.js]: done

**Comment by mathiasbynens** [comment]
I’m gonna go ahead and merge this since this should be fixed asap on the live site. Let’s tackle the tests in a separate PR. Thanks!

**Pull request merged**